internal/cmd/oohelperd/http_test.go

```diff
@@ -142,3 +142,54 @@ func Test_httpMapFailure(t *testing.T) {
 		})
 	}
 }
+
+func TestParseAltSvc(t *testing.T) {
+	resp := &http.Response{
+		Header: http.Header{"Alt-Svc": []string{"h3=\":443\"; ma=3600,h2=\":443\"; ma=3600"}}}
```

Comment:
```suggestion
		Header: http.Header{"Alt-Svc": []string{`h3=":443"; ma=3600,h2=":443"; ma=3600`}}}
```

Trick to worry less about quoting `"`: we use raw strings and the result is a bit more readable

Reply:
Ah!